### PR TITLE
update conformance links, various tweaks

### PR DESCRIPTION
**Pull request by cholmes**
**Related Issue(s):** #

**Proposed Changes:**

1. Update the conformance URI's to reflect where we actually (will) have content.

**PR Checklist:**

- [ ] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [x] This PR has **no** breaking changes.
- [ ] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-api-spec/blob/dev/CHANGELOG.md) **or** a CHANGELOG entry is not required.

### CODE REVIEW

File: core/README.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ The core of a STAC API is its landing page, which is the starting point to disco
         }
     ],
     "conformsTo" : [
-        "http://stacspec.org/spec/api/1.0.0-beta.1/core",
-        "http://stacspec.org/spec/api/1.0.0-beta.1/stac-search"
+        "http://api.stacspec.org/1.0.0-beta.1/core",
```
Comment (m-mohr) [inline core/README.md:33]: It seems there are some places that are missing the `v` before the version number.

File: core/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # STAC API - Core Specification
 
-- **Conformance URI:** <http://stacspec.org/spec/api/1.0.0-beta.1/core>
+- **Conformance URI:** <http://api.stacspec.org/1.0.0-beta.1/core>
```
Comment (m-mohr) [inline core/README.md:3]: Should we go for https? It's the future of the Web...

File: overview.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ column is more of an example in some cases. OGC API makes some endpoint location
 
 STAC API is evolving to utilize OAFeat's 
 '[Conformance](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_declaration_of_conformance_classes)' JSON structure. For 
-STAC API 1.0.0-beta.1 we declare new STAC Conformance classes, and specify which OAFeat ones to use. These are detailed in the relevant
+STAC API v1.0.0-beta.1 we declare new STAC Conformance classes, and specify which OAFeat ones to use. These are detailed in the relevant
```
Comment (m-mohr) [inline overview.md:101]: It's fine to have the `v`, but looks like a unintended side-effect of search&replace.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ The definitive specification for STAC API is provided as an [OpenAPI](http://ope
 contained within several YAML files in the various directories. These live in the same place as the markdown defining
 various parts of the specification. Currently we expect developers to be up to speed with
 OpenAPI and using their own tools to modify things. In the future we will provide tools to make it easier to work with.
-We do publish full OpenAPI files online, at TODO: Matt to add final location.
+There are html version of the OpenAPI files online at <http://api.stacspec.org/>.
```
Comment (m-mohr) [inline CONTRIBUTING.md:79]: By default this page will likely only be an empty HTML page. I guess we should (for now) go for:

```suggestion
There are html version of the OpenAPI files online at `http://api.stacspec.org/{version_number}` with `{version_number}` being the git tag or `dev`.
```

File: README.md [hunk 5]
```diff
@@ -19,7 +19,8 @@ The API can be implemented in compliance with the *[OGC API - Features](http://d
 to search STAC Catalogs, where the features returned are STAC [Items](stac-spec/item-spec/item-spec.md), 
 that have common properties, links to their assets and geometries that represent the footprints of the geospatial assets.
 
-The specification for STAC API is provided as an [OpenAPI](http://openapis.org/) 3.0 specification in addition to human-readable documentation.
+The specification for STAC API is provided as files that follow the [OpenAPI](http://openapis.org/) 3.0 specification, 
+rendered online into HTML at <http://api.stacspec.org/>, in addition to human-readable documentation.
```
Comment (m-mohr) [inline README.md:23]:
```suggestion
rendered online into HTML at <http://api.stacspec.org/v1.0.0-beta.2>, in addition to human-readable documentation.  

```

File: core/README.md (hunk 2 above)
Comment (cholmes) [reply core/README.md]: It's not working yet, which is why I held off. I was thinking of just going to https for the next version. Or if the cert is issued and https propogates we can do it.

Comment (m-mohr) [reply core/README.md]: https is working fine for me: https://api.stacspec.org

File: README.md (hunk 5 above)
Comment (cholmes) [reply README.md:23]: Should this be beta.1?

Comment (m-mohr) [reply README.md:23]: Yes, beta.2 is STAC spec...

File: core/README.md (hunk 2 above)
Comment (cholmes) [reply core/README.md]: Just checked it and it worked for me too. I think it was propogating out. Github before wasn't letting me force https, but it just let me set it, so I did. So yes, let's switch to https. If you can do it go for it, I've got to jump on a call now, but could do it after if you don't get to it.

**Review by m-mohr — APPROVED** [review]

**Pull request merged**